resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -583,8 +599,16 @@ public class ApiClient implements AutoCloseable {
                 @Override
                 public void onCompleted(ApiClientConnectorResponse connectorResponse) {
                     try {
+                        ApiClient.Retry retry = new ApiClient.Retry();
+                        long timeElapsedPerRequestInMills = 0L;
                         ApiResponse<T> response;
                         try {
+                            do {
```

Comment:
I don't think there should be a loop here. This code will invoke the async API infinitely because the connectorResponse variable that's being checked is never updated here.